_episodes/03-lists.md

```diff
@@ -38,20 +38,23 @@ odds are: [1, 3, 5, 7]
 ~~~
 {: .output}
 
-We select individual elements from lists by indexing them:
+We can select individual elements from lists by indexing them.
+Indexing allows us to access individual elements inside a container (such as list or string).
 
 ~~~
-print('first and last:', odds[0], odds[-1])
+print('first and last:', odds[0], odds[3], odds[-1])
```

Comment:
We can no longer say `first and last` as we have three values. What if we split it?
```py
print('first element:', odd[0])
print('last element:', odd[3])
print('"-1" element:', odd[-1])
```

Reply:
Done in a5ba448